src/components/MainPage/SideMenu.tsx

```diff
@@ -21,42 +26,69 @@ const SideMenu = () => {
     },
   ];
 
+  // scorll lock
+  if (isSideMenuOpen) {
+    document.body.style.overflow = 'hidden';
+  } else {
+    document.body.style.overflow = 'unset';
+  }
```

Comment:
안녕하세요 코드 간추리기 위원회에서 나왔습니다~ 

```suggestion
document.body.style.overflow = isSideMenuOpen ? 'hidden' : 'unset';
```